addons/web/static/src/views/form/form_group/form_group.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="web.Form.OuterGroup" owl="1">
+  <div class="o_group" t-attf-class="{{ allClasses }}" t-att-style="props.style">
+    <t t-slot="title" t-if="props.slots.title" />
+    <t t-foreach="getItems()" t-as="item" t-key="item_index" >
+      <br t-if="item.newline" />
+      <t t-slot="{{ item.name }}" className="'o_group_col_' + item.size"/>
+    </t>
+  </div>
+</t>
+
+<t t-name="web.Form.InnerGroup" owl="1">
+  <table t-attf-class="{{ allClasses }}" class="o_inner_group o_group" t-if="props.slots" t-att-style="props.style">
+      <tbody>
+          <td t-if="props.slots.title" t-attf-colspan="{{ props.maxCols }}" style="width: 100%">
```

Comment:
inline style?

Reply:
some tests assert that, I did not want to change it everywhere... but I agree, it should directly be an attr on the node.
That is computed in JS, so I guess it is rather hard to do it in css